CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- The [Conjur Ansible role](https://galaxy.ansible.com/cyberark/conjur-host-identity) has been
+  migrated to this collection, where it will be maintained moving forward.
+  [cyberark/ansible-conjur-host-identity#30](https://github.com/cyberark/ansible-conjur-host-identity/issues/30)
+- Added retries to tasks/identity/Request identity from Conjur.
+  This will increase the reliability of host factory requests without introducing any extra delay 
```

Comment:
is this something you did in this PR, or was this ported from an unreleased change in the role?

Reply:
This was ported from the Conjur-host-identity project. I wasn't sure what to do with it, because those features are currently unreleased.

Reply:
I think we can leave it in there, and tag that project just before we tag this one to indicate the migration

Reply:
just rereading and realizing this is not clear, I meant that we can drop this entry from the collection changelog and just be clear in the collection changelog that we migrated at the point-in-time of the {soon to be added} last role release.

Reply:
Ah gotcha, I will do so!